e2eshark/onnx/models/AlexNet_vaiq_int8/model.py

```diff
@@ -1,5 +1,11 @@
 import numpy, torch, sys
 import onnxruntime
+from PIL import Image
+import torchvision.transforms as transforms
+import requests
+
+def to_numpy(tensor):
```

Comment:
NIT: since it's repeated in quite a few models we should probably move this to commonutils.py and import from there 
Otherwise LGTM!

Reply:
Good point, moved